### PR TITLE
new feature

### PR DESCRIPTION
**Pull request by ChamodAK**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -54,4 +54,4 @@ sonarqube {
 
 tasks.sonarqube.dependsOn jacocoTestReport
 
-//comment
+// newly added
```
Comment (cham2git) [inline build.gradle:57]: change this to "Newly added feature"

Comment (ChamodAK) [reply build.gradle:57]: Error is still there. Please fix it.

Comment (cham2git) [inline build.gradle:57]: Change is still needed to be done

Comment (cham2git) [reply build.gradle:57]: error is still there

**Pull request merged**